aioredis/commands/streams.py

```diff
@@ -209,9 +209,9 @@ def xack(self, stream, group_name, id, *ids):
         """Acknowledge a message for a given consumer group"""
         return self.execute(b"XACK", stream, group_name, id, *ids)
 
-    def xdel(self, stream, id):
+    def xdel(self, stream, id, *ids):
```

Comment:
```suggestion
    def xdel(self, stream, *ids):
```

Just being concise

Reply:
While I do agree with this implementation, the rest of the library is implemented the way that @art1415926535  has done.

We should have a separate effort to move to the above pattern.